### ISSUE TITLE
[Chisel and Bits compatibility] ...

### ISSUE THREAD

**Issue by HydroFrog64** [PlatinPython/RGBBlocks#2]
**Is your feature request related to a problem? Please describe.**
So I saw on the main page that you told someone to open up an issue on github for compatibility for C&B. I think this mod would be the PERFECT companion mod with chisel and bits, especially since Flat Colored Blocks is no longer being supported. The problem is I don't know if it would be even possible for the two mods to work together without both developers doing something together to make it work, just because of the nature of both mods. I have a feeling this is something that will require the most work on C&Bs end.

**Describe the solution you'd like**
For Chisel and Bits (The new one) to be able to chisel away at custom color blocks and be able to place chiseled bits of the colored blocks. Basically full compatibility. The only way I could see this working from your end is to turn every possible block/color combination into a real individual block which sounds absolutely ridiculous so I won't even bother suggesting it. My suggestion would be to contact the C&B mod author directly. (here's a link to the community run C&B discord: https://discord.gg/SMYAdKYm and here's a link to the official discord: https://discord.com/invite/RrnACqpzP5 )

**Describe alternatives you've considered**
Well like I said I imagine the main bulk of the work would be C&B making itself compatible with your mod than the other way around. I think you could potentially work together with the C&B team if they are okay with it (assuming you also aren't busy)

**Additional context**
This is just a desperate plea for help since Flat Colored Blocks is dead and I REALLY want something that can replace it. Thank you for your time and patience.

**Comment by fbm5733** [comment]
Hi, I know this has been open a while, and I'm not sure if there's any intent to implement this, but I wanted to follow up with some details I discovered on this while I was trying myself to get the two working. It's possible the mod author already knows about this but figured I'd say something anyway.

Chisel and Bits has a tag 'chiselsandbits:chiselable/forced' that, using datapacks, I was able to give to all of the full blocks from RGB Blocks (not stairs, slabs, etc, as those already never work with C&B).

Doing this made the blocks chiselable, which is great, but sadly the blocks do not retain their color after being chiseled. So this seems to be the main issue with the mod compatibility, the chisels aren't saving the color value anywhere. That's where I got stuck, as I don't think there's anything that can be done about that without modifying the mod itself (one or both).

This was done in 1.16.5 by the way, as that's the version I would personally want to use this in. Just thought I'd share my findings, even if it won't go anywhere.

**Comment by RecoveryDeer** [comment]
Bump! I'd love to see this for the mod.

**Comment by RecoveryDeer** [comment]
yeah I'd no joke literally pay yall for chisel & bits or littletiles support <333 just give me the dono link and im on it
